int8 enable_x_dir
int8 enable_y_dir
int8 enable_yaw_dir
geometry_msgs/Pose2D pose
int8 stop
int8 state
int8 type